components/Layouts/ContactForm/index.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import styles from './styles/ContactForm.module.scss'
+import { string } from 'prop-types'
+
+const ContactForm = ({ email }) => {
+  return (
+    <form action={email} method='POST' className={styles.ContactForm}>
+      <input type="hidden" name="_captcha" value="false"/>
+      <h2>Susisiekime!</h2>
+      <div className={styles.row}>
+        <div className={styles.col}>
+          <div className={styles.inputBox}>
+            <input type="text" name="name" required="required"/>
+            <span className={styles.text}>Vardas</span>
+            <span className={styles.line}></span>
+          </div>
+        </div>
+        <div className={styles.col}>
+          <div className={styles.inputBox}>
+            <input type="text" name="email" required="required"/>
+            <span className={styles.text}>Email Adresas</span>
+            <span className={styles.line}></span>
+          </div>
+        </div>
+      </ div>
+      <div className={styles.row}>
+        <div className={styles.col}>
+          <div className={styles.inputBox}>

```

Comment:
truksta label (del accessibility)

check this out https://webaim.org/techniques/forms/controls

Reply:
Siaip dar pasidomek butinai apie accessibilty nes tai lb svarbu. Labai daug visokiu niuansu yr